<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="welcomelbl">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>..</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Search</string>
   </property>
   <widget class="QPlainTextEdit" name="inputTxt">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>By</string>
    </property>
    <widget class="QRadioButton" name="author_rbut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Author</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="title_rbut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="year_rbut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Published Year</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="searchBut">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="borrowBut">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Borrow Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkBut">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>See All Borrowed Books</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>691</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
